src/constants.ts

```diff
@@ -24,5 +24,5 @@ export const homeUrl = "/";
 /**
  * Maximum number of items to show in a paginated view.
  */
-export const defaultPageSize = 1;
+export const defaultPageSize = 10;
```

Comment:
Can we do more than 10? For example, 50 or 100?

Reply:
We can add switch later on, for now I'll set 50.